app/src/main/kotlin/com/simplemobiletools/camera/implementations/CameraXPreviewListener.kt

```diff
@@ -0,0 +1,17 @@
+package com.simplemobiletools.camera.implementations
+
+import android.net.Uri
+
+interface CameraXPreviewListener {
```

Comment:
Added this so the `CameraXPreview` is not tightly coupled with the `MainActivity`